data/language/english_uk.txt

```diff
@@ -3945,6 +3945,19 @@ STR_5603    :Guest has left ride
 STR_5604    :Guest has bought item
 STR_5605    :Guest has used facility
 STR_5606    :Guest has died
+STR_5607    :RCT1 Classic
+STR_5608    :Corkscrew Follies
```

Comment:
Change to Added Attractions as that is the correct name for UK. Other languages like US can use CF if they want to.